cypress/e2e/wards.cy.js

```diff
@@ -1,41 +1,40 @@
 describe("wards spec", () => {
   it("passes", () => {
-    cy.visit("/municipalities/CPT/wards/cpt-ward-1/");
+    cy.visit("/municipalities/CPT/wards/cpt-city-of-cape-town-ward-1/");
   });
 });
 
-describe("nav tray visible on hover", () => {
-  it("passes", () => {
-    cy.visit("/municipalities/CPT/wards/cpt-ward-1/");
-    cy.get(".tray-tabs").first().parent().invoke("show");
-    cy.get(".tray-tabs").first().trigger("mouseover");
-    cy.get(".nav-tray").invoke("show");
-    cy.get(".nav-tray").should("be.visible");
-  });
-});
+// describe("nav tray visible on hover", () => {
```

Comment:
Don't commit commented out code please @cyber237 @Hazmid

Reply:
I would remove it now

Reply:
Ok, I will remove them.